src/PackageServiceProvider.php

```diff
@@ -66,7 +66,7 @@ public function boot()
 
             if ($this->package->hasTranslations) {
                 $this->publishes([
-                    $this->package->basePath('/../resources/lang') => resource_path("lang/vendor/{$this->package->shortName()}"),
+                    $this->package->basePath('/../resources/lang') => lang_path("lang/vendor/{$this->package->shortName()}"),
```

Comment:
I think `lang` prefix should be removed from the path string since it's automatically added by Laravel:

```php
lang_path('foo');
// => "/Users/theraloss/laravel9/lang/foo"
```